cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -2937,6 +2938,27 @@ pub(crate) fn emit(
         Inst::EpiloguePlaceholder => {
             // Generate no code.
         }
+
+        Inst::ElfTlsGetAddr { ref symbol } => {
```

Comment:
Is there something meaningful we can add as a comment, explaining what the sequence does?
Also: can it be reframed as other Vcode::Inst instructions that would be recursively generated? This would act as documentation, and help understand and maintaining this sequence a bit better.

Reply:
The thing is that it needs this exact byte sequence. Any tiny change will result in a linker error as the linker doesn't know how to rewrite it anymore. For the old backend I wrote a few comments that you could use here: https://github.com/bytecodealliance/wasmtime/blob/f579d088baac9df6bb852ecc19d8216fb94438a1/cranelift/codegen/meta/src/isa/x86/recipes.rs#L3356-L3377

Reply:
Added comments similar to the original x86-backend ones, as well as a comment noting that the linker expects exactly these bytes.